docker/gcb/fuzzer.yaml

```diff
@@ -36,12 +36,12 @@ steps:
     '--tag',
```

Comment:
Please review the changes here if you get a chance. I think these are correct but it would be easy to miss an occurrence.

Reply:
Will do